### DISCUSSION TITLE
Getting start and end line number of each node in the DotDiff output

### DISCUSSION THREAD

**Discussion by mahtab-nejati** [GumTreeDiff/gumtree#358] [Q&A]
I know that the getDotLabel method returns the label for Dot output nodes. Is there a way I could access the start and end line in the source code file for a node (maybe representing a single line of code or a multiline block of code) within the toString method so that it gets passed to this label?

**Comment by jrfaller** [comment]
Hi! Do you mean the line in the corresponding source code file or dot file? Cheers!

**Reply by mahtab-nejati** [reply]
I meant the line in the corresponding source code file.

**Comment by jrfaller** [comment]
You have the absolute begin and end position directly on a Tree (getPos(), getLenght()). Cheers!

**Reply by mahtab-nejati** [reply]
Thank you for the response. I was hoping there would be a way to get the line number of the element within the source file without referring to the source file and locating the line using the values returned by these methods.
